### ISSUE TITLE
Feature request - Fade

### ISSUE THREAD

**Issue by ChuzaWick420** [daxcore/mkdocs-obsidian-interactive-graph-plugin#3]
Hi, firstly, I want to mention that this project seems to be very cool. 
I was looking for something like this and it does exactly what I wanted except for some few things which annoy me.

As you can see in the image, as your vault grows, it becomes a mess and is not really useable. 😅
![image](https://github.com/user-attachments/assets/db8993df-b5a6-4778-bf0c-aefd02daad36)

If somehow, a fade effect is added for the text, as it is in obsidian, then it can become a lot more useable :)
![image](https://github.com/user-attachments/assets/0845250d-7a14-4981-9d04-190b210a5720)

**Comment by daxcore** [comment]
hey. u could try to configure your graph with e.g.:

* disable the label strings via:
```
        label: {
          show: false
        },
```
* hide overlap option:
```
        labelLayout: {
          hideOverlap: true // true could be a good idea for large graphs
        },
````
* reduce size of edges via `node.symbolSize`
* or try to find good setup for u via the live echart web demo: https://echarts.apache.org/examples/en/editor.html?c=graph-force

**Comment by ChuzaWick420** [comment]
I did tested the experimental version and found some limitations.
The graph does not appear when you are using the absolute paths.
It does look very clean though :D

- `[[Test_Doc]]` (works)
- `[[docs/Test_Dir/Test_Doc]]` (doesn't work)

**Comment by daxcore** [comment]
is `[[docs/Test_Dir/Test_Doc]]` really something obsidian would create and support? it tried to handle the default obsidian paths.

**Comment by daxcore** [comment]
initial issue fixed by release 0.3.0. added an option to minimize the graph inside the sidebar to related edges of current page only.
